test/metabase/pulse/test_util.clj

```diff
@@ -145,7 +145,7 @@
    [{:name "State",
      :slug "state",
      :id "63e719d0",
-     :default ["CA", "NY"],
+     :default ["CA", "NY", "NJ"],
```

Comment:
added an extra value here to more fully test the logic in the `formatted-list` helper above